### PR TITLE
open issues

### PR DESCRIPTION
**Pull request by mce777**
This PR address the following open issues:

Missing attributes: https://github.com/mce777/currStyles/issues/5

Added `.editorconfig`: https://github.com/mce777/currStyles/issues/9

Added README. https://github.com/mce777/currStyles/issues/11

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,2 @@
+##Currency Component Styles
+description pending....
```
Comment (MartinHelmut) [inline README.md]: Description pending? This is not a resolved task :P

File: lib/components/molecules/input_dropdown/InputDropdown.js [hunk 2]
```diff
@@ -6,25 +6,29 @@ function InputDropdown(props) {
...
 				handleChange={props.handleInputChange}
+				value={props.inputLabelValue}
+				inputName={props.inputName}
 			/>
 			<Dropdown
 				name={props.name}
 				value={props.value}
 				options={props.options}
 				handleChange={props.handleDropdownChange}
+			    selectName={props.selectName}
```
Comment (MartinHelmut) [inline lib/components/molecules/input_dropdown/InputDropdown.js]: Wrong indention

Comment (mce777) [inline lib/components/molecules/input_dropdown/InputDropdown.js]: Ok, I fixed it 👍

File: README.md (hunk 1 above)
Comment (mce777) [inline README.md]: I have now resolved the task.

File: README.md [hunk 3]
```diff
@@ -0,0 +1,2 @@
+##Currency Component Styles
+Description is no longer pending.
```
Comment (MartinHelmut) [inline README.md]: 1. How to install
2. How to build
3. How to debug
4. How to test
5. Author

Comment (mce777) [inline README.md]: Is there a testing/debugging process beyond the visual inspection of the components in the living styleguide?

Comment (mce777) [inline README.md]: Ah man, I should have just written "description text in a later release". Live and learn.

Comment (mce777) [inline README.md]: Updated README. https://github.com/mce777/currStyles/pull/12

**Pull request merged**